src/main/java/uk/ac/ebi/ega/egafuse/validator/Validator.java

```diff
@@ -0,0 +1,96 @@
+/*
+ *
+ * Copyright 2020 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ega.egafuse.validator;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.StringTokenizer;
+
+import jnr.posix.util.Platform;
+import joptsimple.OptionSet;
+import joptsimple.internal.Strings;
+
+public class Validator {
+
+    public static boolean isValid(OptionSet optionSet) {
+        try {
```

Comment:
If the parser is properly configured almost all these checkings can be removed.

Reply:
Some of the conditions are removed but still this class is useful to check other conditions.